berkeley-function-call-leaderboard/model_handler/gpt_handler.py

```diff
@@ -1,32 +1,37 @@
+import json
+import os
+import time
+
+from model_handler.constant import (
+    GORILLA_TO_OPENAPI,
+    GORILLA_TO_PYTHON,
+    SYSTEM_PROMPT_FOR_CHAT_MODEL,
+    USER_PROMPT_FOR_CHAT_MODEL,
+)
 from model_handler.handler import BaseHandler
 from model_handler.model_style import ModelStyle
 from model_handler.utils import (
-    convert_to_tool,
-    convert_to_function_call,
+    ast_parse,
     augment_prompt_by_languge,
+    convert_to_function_call,
+    convert_to_tool,
     language_specific_pre_processing,
-    ast_parse,
-)
-from model_handler.constant import (
-    GORILLA_TO_OPENAPI,
-    GORILLA_TO_PYTHON,
-    USER_PROMPT_FOR_CHAT_MODEL,
-    SYSTEM_PROMPT_FOR_CHAT_MODEL,
 )
 from openai import OpenAI
-import os, time, json
 
 
 class OpenAIHandler(BaseHandler):
-    def __init__(self, model_name, temperature=0.7, top_p=1, max_tokens=1000) -> None:
+    def __init__(self, model_name, temperature=0.0, top_p=1, max_tokens=1000) -> None:
         super().__init__(model_name, temperature, top_p, max_tokens)
         self.model_style = ModelStyle.OpenAI
         self.client = OpenAI(api_key=os.getenv("OPENAI_API_KEY"))
 
-    def inference(self, prompt,functions,test_category):
+    def inference(self, prompt, functions, test_category):
```

Comment:
Shall we overwrite it in FireworksAIHandler? If we publish our results, we will likely need to send a PR against upstream. I would prefer to keep the changes in the fireworks-specific code.